Makefile

```diff
@@ -250,6 +250,14 @@ cross:
 $(notdir $(abspath $(wildcard cmd/*/))): generated_files
 	hack/make-rules/build.sh cmd/$@
 
+# Add rules for kube-scheduler
+#
+# Example:
+#   make kube-scheduler
+.PHONY: kube-scheduler
+kube-scheduler: generated_files
+	hack/make-rules/build.sh plugin/cmd/kube-scheduler
```

Comment:
I saw that there is only `kube-scheduler` in `plugin/cmd/`, so I just put it directly here instead of the way you mentioned. Surely your way is compatible.

Is it OK as below?

```
.PHONY: $(notdir $(abspath $(wildcard plugin/cmd/*/)))
$(notdir $(abspath $(wildcard plugin/cmd/*/))): generated_files
	hack/make-rules/build.sh plugin/cmd/$@
```

Thanks!